homeassistant/components/blastbot_cloud/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I don't believe this file needs to be committed.

Reply:
It doesn't need to be committed, but we allow it since it's generated by the translations develop script and to help testing. The other language translation files should not be included.